redhat-operators/tektoncd-pipeline-operator/tektoncd-pipeline-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,250 @@
+          "metadata": {
+            "name": "example-install",
+            "namespace": "tekton-pipelines"
+        },
+          "spec": {}
+        }
+      ]
+  name: tektoncd-pipeline-operator.v0.0.1
+  namespace: tekton-pipelines
+spec:
+  version: 0.0.1
+  maturity: alpha
+  provider:
+    name: Red Hat, Inc.
+  maintainers:
+  - name: Red Hat, Inc.
+    email: customerservice@redhat.com
+  icon:
+  - base64data: /9j/4AAQSkZJRgABAQIAdgB2AAD/2wBDAAMCAgICAgMCAgIDAwMDBAYEBAQEBAgGBgUGCQgKCgkICQkKDA8MCgsOCwkJDRENDg8QEBEQCgwSExIQEw8QEBD/2wBDAQMDAwQDBAgEBAgQCwkLEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBD/wAARCAAoACgDAREAAhEBAxEB/8QAGQAAAwEBAQAAAAAAAAAAAAAABQYHCAIE/8QAKxAAAQQBAwMEAgEFAAAAAAAAAQIDBAUGBxESABMhFBUiQTEyYSNRUnGR/8QAGgEAAwEBAQEAAAAAAAAAAAAAAwQFBgcBAv/EAC8RAAEDAwMDAgUDBQAAAAAAAAECAxEABCEFEjFBUWETIiMycYGhFJGxQtHh8PH/2gAMAwEAAhEDEQA/ANpZbStz7zN8tmX1pCtMbso/YRGmrZLkRTDfBpAHjkpRVxOx3X4O4J6o/qNjSGwBBH5mr9mk/CaCQQsHkcEE5PgdfFFno1/Bxxqdq3q3IpyppRi17ElmE6pPniH3U7Kcc22CuHFIP/epzriE54py3Ql5/wBPTLb1M5UQVDzA4A7TJryYkmbd483J0w1ocevvTFTldYzUT2Vr2+0q3cbP8gkD7H10jvLo+Av3duaq3iG7G526xYfBn5kpKD+49p+n5pNeppqqxvUKxyK+cyiPk9fTR4UueVLYfElAeStCNkfNJWQlI4hvY+SSehtME/EUTunvVZ/Um0rOmstIDBbWslKcEFJ2wTJwYkkzumtUdUq5TUrznHIL+r+HzGw6HJqJLsxlKtm5XpEc2OQ+1JW5uD14VxCa0enOqGl3E/07QD1G8wr9wKyXqJ79k+QWkzKa/IG8qcdW6mM9GPZDQc2DTY23CUo2Vy34+CNvs566cW6opg7q7loLdrpts2bdTZt4AkHMxyfqcRz/ABSlWxrOiksS48bI2Mm7pcq0V8YkpKAClZ8brClniQk+NiTv+OiWlqUnceaJq2qN3SVNygsxCtx79PEDOefzWuMfpXbzXDGLjK4UiJPlYm1fya0nZpu1bUlgurR/mlDpSP7bfx1dCMbq4hc3YZ0523tzKQspCuuw+6J7SKvvXtZWlPUTA2c4rY3YlCHa1b3qq6WU8g27tsUqAIJQoeFDff7HkDoL7XqpgGCOKtaJq50l5RUnc2sQpPcdwehHIqB5LnjlpClxZeF2rthAdfo7BLzT9tTvMFYDy0lB74SFtpIKR4I4k9BQCvkZ47itk3aC2UFNvJ2KhxMbW3AYO0HG2YJmfrS5Q6g12A0/fdw23eaqE+kqvYoMioiSXJBR3XHnZCg7vyQ2DxBSAN/9Ps28wBikdRSu9cJLglWVbyFkbZgAJx1PmnSlyGHpTkt1Pk1asu1EsW44s41fKCIVLFU6hDUcyHyT+7qCo/JaiQogDbptLBeSMwn+azd04bptKEna2OJ5J7wP+Cqfges8PK8ifw2+x+VQXjSngwlbgfhz+0QHvTSUgJdLZOy07BSfPjwdvl6zU236qTKfyJ4kdJ6VIWkJ4M0f1JzePp/ij9+4htx5TrUSI26opQt91QSgLUP1SCd1H6APSZpvTrM31wGpgZJ+gyY89vNRrHNWK3EdIH6dqzcF5GsJVf7o3BdMBUt2coFaZBSWv1cCxur8dEaZJgVa1C2Nzf8ArEewhJiRMBIwRz0jiuM81dobjRzN8csbN+wsRIsaNqwYguOxklb5RFU5IQntI8La8lQ/Hnp23tV+skxjB8+cVKDRafQ4kQMGPtmgmgul+MajybdWVlu6qMUmrrfRrmGXHsbRTaFyrJSyAVpdBaKEElI2J2322avHl2oTswVCZ4IHQfbrQH3yo4/0dBVKx7Q69pMgoIhyqGrDsRtJNtSQUQ1ic0p5DqBHW+VkFlHec2ASFEcQT48hXqDa21nb71gAmcYjMRyYpFQUpUk0Q1nq8tuZ+PRqupu5lNEfVPleyuRUyvVtLQqOCZCgA2PmTx8kgA+D1OQAeTVfTnmmUOFRG4iBMxBmeOvFRm3z2shY7ndbcIVTWNpPM6PiyqST3Y8uM42Wv6jSVtOGQGUFXE8QV7gnz1RZtlKUkpyO8jr/AGorp9yFAzAjdPIP5xOKHPao0trhcOPGhOC2eyNzIJOFvUEtx6a+qUe1AUeCWkcE9tXIkp5oG/xB3fbslpcJPyxG6RAxz3/xUt5zarviKrWjFTn1XqNk06ZjeS1eN3zXuUhF85AU4iz5IbSI/pVKIa7KQOKv17adj5PSd+phVugJUkrTj27vl8z1nt3pVJUVZGKt/UeiV//Z
+    mediatype: image/png
+  links:
+  - name: Tekton Pipeline GitHub Repository
+    url: https://github.com/tektoncd/pipeline
+  - name: Tekton Pipeline Operator GitHub Repository
+    url: https://github.com/tektoncd/pipeline
```

Comment:
this needs to point to the actual operator repo

Reply:
fixed
https://github.com/openshift/tektoncd-pipeline-operator